server/main_server/src/irods_server_control_plane.cpp

```diff
@@ -367,20 +367,20 @@ namespace irods
         rodsEnv my_env;
         _reloadRodsEnv( my_env );
 
-        int re_pid = 0;
+        int delay_server_pid = 0;
         // no error case, resource servers have no re server
```

Comment:
re -> delay

and also, this comment is not always true now -- we are separating the functionality via delay server migration...

Reply:
removed comment